cdap-docs/admin-manual/source/operations/cdap-ui.rst

```diff
@@ -38,5 +38,234 @@ and are confident of their suitability and compatibility. Earlier versions (and
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "favicon-path": "/cdap_assets/img/favicon.png"
+
+
+   * - ``footer-text``
+     - No attributes
+     - Text content to displayed on the footer component.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "footer-text": "Licensed under the Apache License, Version 2.0"
+
+   * - ``footer-link``
+     - No attributes
+     - Link to route to when user clicks on footer text.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "footer-link": "https://www.apache.org/licenses/LICENSE-2.0"
+
+   * - ``dashboard``
+     - No attributes
+     - Whether to show 'Dashboard' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "dashboard": true
+
+   * - ``reports``
+     - No attributes
+     - Whether to show 'Reports' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "reports": true
+
+   * - ``data-prep``
+     - No attributes
+     - Whether to show 'Preparation' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "data-prep": true
+
+   * - ``pipelines``
+     - No attributes
+     - Whether to show 'Pipelines' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "pipelines": true
+
+   * - ``analytics``
+     - No attributes
+     - Whether to show 'Analytics' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "analytics": true
+
+   * - ``rules-engine``
+     - No attributes
+     - Whether to show 'Rules Engine' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "rules-engine": true
+
+   * - ``metadata``
+     - No attributes
+     - Whether to show 'Metadata' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "metadata": true
+
+   * - ``hub``
+     - No attributes
+     - Whether to show 'Hub' feature on the nav bar.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "hub": true
+
+   * - ``ingest-data``
+     - No attributes
+     - Whether to show 'Ingest Data' feature in the Preparation app.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "ingest-data": true
+
+
+   * - ``add-namespace``
+     - No attributes
+     - Whether the user can add a new namesace in the UI.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "add-namespace": true
+
+.. highlight:: xml
+
+However, it is not recommended to overwrite values in ``default.json`` for customizations, since these values will be reverted when CDAP is upgraded. Instead of doing this, the user is encouraged to create their own theme file using this spec, and link to this file. To do so, add this property to ``cdap-site.xml``::
```

Comment:
Also, since this is a separate paragraph with no context from any previous text, do not start with ``However,``